myfile.js

```diff
@@ -1,3 +1,16 @@
-// my homework2
-// my homework3
-console.log('hello world');
+'use strict';
+
+let x : Number = 10;
```

Comment:
В JS не має типів, це звідки взято?)

Також всі змінні, що не перезаписуються в подальшому виконанні скрипта, треба об'являти через константи.
```suggestion
const x = 10;
```

Reply:
І так далі

Reply:
> В JS не має типів, це звідки взято?)

з лекції)
 але потім зрозуміла, що тип автоматично відображався у webstorm.

Reply:
О, хороший пойнт, до речі, дякую. Буду про це говорити явно, бо може бути не очевидно.